### PR TITLE
Exclude non-OCI refs from list of OLM image refs

### PR DESCRIPTION
**Pull request by lcarva**
Ref: EC-874

### CODE REVIEW

**Review by zregvart — COMMENTED** [review]
Perhaps this belongs to `image.parse`

File: policy/release/olm.rego [hunk 1]
```diff
@@ -348,6 +348,7 @@ all_image_ref(manifest) := [e |
 		],
 	]
 	some i in imgs
+	not contains(i.ref.repo, "://")
```
[the remark this answers is not in the document]
Comment (lcarva) [reply policy/release/olm.rego:351]: Yeah, definitely. I'll update.

[the remark this answers is not in the document]
Comment (lcarva) [reply policy/release/olm.rego]: Done. Let me know what you think.

**Review by zregvart — APPROVED** [review]
Nice :+1:

File: policy/lib/image.rego [hunk 2]
```diff
@@ -23,6 +23,8 @@ parse(ref) := d if {
 		),
 	)
 
+	not contains(repo, "://")
```
Comment (simonbaird) [inline policy/lib/image.rego:26]: I'm assuming `oci://` is not a thing in this context, right? Or `docker://`.

Comment (zregvart) [reply policy/lib/image.rego:26]: I haven't seen that being used outside of skopeo, so I'm not expecting it here

Comment (lcarva) [reply policy/lib/image.rego:26]: Agree.

**Pull request merged**